_component_design/button-bars.md

```diff
@@ -11,7 +11,7 @@ resource: true
 last-modified: 2018-09-20
 helix-ui-css: true
 helix-ui-javascript: na
-pagelink: https://rackerlabs.github.io/helix-ui/components/menus/#split-menu
+pagelink: https://rackerlabs.github.io/helix-ui/components/menu/#split-menu
```

Comment:
Shouldn't this point to `https://rackerlabs.github.io/helix-ui/components/button`? (a split menu _is_ a button bar, but not all button bars are split menus)

Reply:
I believe this was from when we had a lot of confusion around terms. I'd check what it points to with the designers to be sure, but it might be worth letting be until we get to thunderdome.